Remove test setting and add model to one_click setting

stuartjennings-up opened the pull request.

stuartjennings-up committed:
Remove test setting and add model to one_click setting

stuartjennings-up committed:
new line end of file

craiga reviewed (commented):
Is this fixture ever likely to be loaded outside of a test? If so, what's the effect of changing the primary key on this object should one already exist in the database?

stuartjennings-up commented:
@craiga I actually didn't change it, I just deleted the test setting that had pk: 125 and added pk: 124 to the one_click setting above it that previously didn't have a pk. I'm not sure if there's some way that these pk's are automatically generated or if they're just added manually like this- will have to look into it.
